README.md

```diff
@@ -56,88 +39,17 @@ This list contains workflows covered by packit tool and links to the documentati
 - [Create a SRPM from the current content in the upstream repository.](https://packit.dev/docs/cli/srpm/)
 - [Sync content of the Fedora dist-git repo into the upstream repository.](https://packit.dev/docs/cli/sync-from-downstream/)
 
-## Configuration
-
-Configuration file for packit is described [here](http://packit.dev/docs/configuration/).
-
-TL;DR
-
-```yaml
-specfile_path: packit.spec
-synced_files:
-  - packit.spec
-upstream_package_name: packitos
-downstream_package_name: packit
-```
-
-## User configuration file
-
-User configuration file for packit is described [here](http://packit.dev/docs/configuration/#user-configuration-file).
-
 ## Requirements
 
 Packit is written in python 3 and is supported only on 3.6 and later.
 
-When packit interacts with dist-git, it uses `fedpkg`, we suggest installing it:
-
-```bash
-sudo dnf install -y fedpkg
-```
-
 ## Installation
```

Comment:
This needs to be updated on the docs part as well -- https://github.com/packit/packit.dev/issues/197

(If you want, you can move this part from here and add it to the docs. [Here](https://github.com/packit/packit.dev/blob/master/content/docs/guide.md) is the relevant markdown document. It will be really nice of you..;)

Otherwise, we can use the following link for now, and update later:
https://packit.dev/docs/guide/#have-packit-tooling-installed-locally

Reply:
Ok I'm gonna read the links and see what to do, thanks.

Reply:
Thanks.